src/components/Article/Article.jsx

```diff
@@ -1,6 +1,68 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Article.css';
 
-// Create an Article function returning the HTML of article block
+function Article() {
+  return (
```

Comment:
```suggestion
const Article = () => (
```
here and in other places can be arrow function.